lego/apps/companies/tests/test_company_interest.py

```diff
@@ -74,6 +81,26 @@ def test_list_with_bedkom_user(self):
         self.assertEqual(company_interest_list_response.status_code, status.HTTP_200_OK)
         self.assertTrue(len(company_interest_list_response.json()["results"]))
 
+    def test_export_to_csv_with_abakus_user(self):
+        AbakusGroup.objects.get(name="Abakus").add_user(self.abakus_user)
+        self.client.force_authenticate(self.abakus_user)
+        company_interest_export_responce = self.client.get(
+            _get_export_company_interest()
+        )
+        self.assertEqual(
+            company_interest_export_responce.status_code, status.HTTP_403_FORBIDDEN
```

Comment:
You should test for both cases, so a 200 response as well.

Reply:
I couldn't manage to find out how make a request in the reverse() function that includes the query parameters "year" and "semester". Could you give me some tips.

Reply:
I don't think you can make a new reverse function for that, but you shouldn't need to. The reverse function is based on what is defined in the urls, but to send a valid response you only need to manipulate the query params (from the view).

If you take the reverse url you have already used and append "?year=2023&semester=whatnot", from a Bedkom-user you should get a valid response.  
You have some discussions of different ways to do it here https://stackoverflow.com/questions/4995279/including-a-querystring-in-a-django-core-urlresolvers-reverse-call